encoding/video-artifacts.md

```diff
@@ -58,8 +58,25 @@ disaster.
 DCT block based video compression has come a long way. If you happen
 to be encoding an MPEG2 TV station or a DVD from a previous decade, you
 will likely come across something like this:
+![Blocky Compression](images/blocky2.png)
```

Comment:
I believe we want to separate images (that are used like figures) with an empty from paragraphs.

Reply:
It is actually separated, it's just the way that commit chose to show the change. The space above it was considered part of the linefeeds commit. You can see the final file [here](https://github.com/Irrational-Encoding-Wizardry/guide.encode.moe/pull/8/files#diff-05e4281dac32bbfc72f55d339784eda4R67).